File: Python/Trzy Kubki.py
n = int(input())
q = int(input())
cups = [number + 1 for number in range(n)]
for _ in range(q):
    i, j = map(int, input().split())
    cups[i -1 ], cups[j - 1] = cups[j -1], cups[i - 1]
print(' '.join(map(str, cups)))

File: C++/Trzy Kubki.cpp
#include <vector>
#include <iostream>

int main() 
{
    int n, q;
    std::cin >> n >> q;
    std::vector<int> cups(n);
    for (int number = 0; number < n; ++number)
        cups[number] = number + 1;
    for (int i = 0; i < q; ++i) 
    {
        int j, k;
        std::cin >> j >> k;
        std::swap(cups[j - 1], cups[k - 1]);
    }
    for (int i = 0; i < n; ++i)
        std::cout << cups[i] << ' ';
    std::cout << '\n';
    return 0;
}